# Get a servo encoder filter's observed servo position (not filtered). 
int8 servo_id
---
int16 position
bool is_valid
